test/ventilation-test.h

```diff
@@ -0,0 +1,8 @@
+/*
+ * ventilation-test.h
+ *
+ */
+
+void startVentilation_openInspiratoryValve();
```

Comment:
consider prefacing all unit tests with `test_...()`. So this would be `test_startVentilation_openInspriatoryValve()`. This lets you easily distinguish between test functions and other helper functions that might be floating around.

Reply:
all the function in a xxx_test file should be tests so I will not do this. 
Moreover all the tests are clearly recognizable by their name "functionToTest_condition_expectedbehaviour"